index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Porfolio - PDM</title>
+    <link rel="stylesheet" href="index.css">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+</head>
+<body>
+<div class="page1">
+    <header>
+    <h1><a href="index.html">Philip Modinos</a></h1>
+    <nav>
+        <ul>
+            <li><a href="portfolio.html">Portfolio</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="contact.html">Contact</a></li>
+        </ul>
+    </nav>
+    </header>
+
+    <aside>
+        <ul>
+            <li><a href="https://github.com/Ninjaphil24"><i class="fa fa-github fa-3x" aria-hidden="true"></i></a></li>
+            <li><a href="https://twitter.com/PModinos"><i class="fa fa-twitter-square fa-3x" aria-hidden="true"></i></a></li>
+            <li><a href="https://www.linkedin.com/in/philip-modinos-14195021"><i class="fa fa-linkedin-square fa-3x" aria-hidden="true"></i></a></li>
+        </ul>
+    </aside>   
+
+    <section class="home">
+        <div class="text">
+        <h2>Hey There. <br> I'm Phil</h2>
+        <h3>I'm a Software Developer</h3>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+        </div>
+        <div class="image">
+            <img id="Vector" src="images/fullimage.png" alt="">
+        </div>
+    </section>
+</div>
```

Comment:
- [ ] It will be nicer if to follow the design from figma. Since you are using template 2 follow the layout, font family,  positioning etc.

Your Design | Figma Design |
:---------------:|:-------------------:|
![a2](https://user-images.githubusercontent.com/39523312/148217600-38c3dc9c-cbc1-4242-91d7-18935b59c477.png) | ![a1](https://user-images.githubusercontent.com/39523312/148217614-24d00a66-15ef-4a4e-a14e-29e8e57203c0.png)

Reply:
Hello Muhammad, thank you for your comments.  I'm going to do all that.  In order not to lose what I've done, I'll use a mediaquery to do the mobile version.

Reply:
Hello again,

I have resolved all the issues, I hope.
I've done something rather complicated, in order not to lose my desktop version, which I worked on for about 4 hours.  

I've created a mobile version using media queries and I've created two versions of that, as when it gets very tight, the image doesn't look so good, so I make it disappear.  

Once the project progresses and javascript is used for the Hamburger menu I will change the media queries to be something simpler.